Packs/Campaign/Layouts/layoutscontainer-Phishing_Campaign.json

```diff
@@ -14,7 +14,7 @@
                     {
                         "h": 1,
                         "hideName": true,
-                        "i": "qpnefeci5o-vvoh19d1ue-dad35280-b1a3-11eb-a261-1f85bcf500af",
+                        "i": "qpnefeci5o-field-changed-qpnefeci5o-pfjeh5flj3-qpnefeci5o-vvoh19d1ue-dad35280-b1a3-11eb-a261-1f85bcf500af",
```

Comment:
@idovandijk what is this

Reply:
Server don't know why it happened, but the `i` property is used for the client as a unique ID, so it's meaningless.